[cleanup] Update contrib/debian/copyright

maflcko opened the pull request:
Splitting this from the other cleanup PR so it can be merged on it's own.

Commit:
Update contrib/debian/copyright

jonasschnelli commented on contrib/debian/copyright:
```diff
@@ -23,60 +23,50 @@ Files: debian/manpages/*
+License: MIT
+Comment: Inspired by Stephan Hutchings Typicons
+
+Files: src/qt/res/icons/tx_mined.png
```
here we should add src/qt/res/src/mine.svg

MarcoFalke replied:
What is the scope of `debian/copyright`? Used for debian bitcoin repos? Which folders should be included in `debian/copyright`?
Actually there is missing more than just `mine.svg` (c.f. https://github.com/bitcoin/bitcoin/blob/88f856a9e78a1440d9fdd8f9d5aa3cd034d66bf9/doc/assets-attribution.md#other )
Is it required to duplicate those files?

MarcoFalke replied:
ping @TheBlueMatt to clarify the scope?

TheBlueMatt replied:
It should include the copyright of anything which is installed, I guess? If the license does not effect the binaries or the files installed directly (https://github.com/bitcoin/bitcoin/blob/master/contrib/debian/bitcoin-qt.install), then it can be skipped I suppose.

jonasschnelli commented:
ACK (see point above).

fanquake commented:
ACK

TheBlueMatt commented:
MIT != Expat. They are not interchangeable.

jonasschnelli commented:
Good point. I think we should remove the whole "Expat" mentioning L70-L72. Everything has moved to MIT.

TheBlueMatt commented:
Oops, sorry. I had two of the MIT's confused. We are all Expat, afaik (MIT is ambiguous).

MarcoFalke commented:
MIT == Expat == content of ./COPYING

Though, I could change to Expat, if that is wanted.

TheBlueMatt commented:
@MarcoFalke Expat is one of several MIT licenses, so it is probably best to be clear which one you are specifying.

MarcoFalke commented:
Force pushed those changes:
* Don't mention the source SVGs
* Call it "MIT/Expat"

wallclockbuilder commented:
ACK

laanwj commented:
Not happy with the duplication, maybe we should simply move to debian's copyright file format and remove our own assets-attribution.md.
That said, ACK

MarcoFalke commented:
+1. Added a commit to merge `doc/assets-attribution.md` into `contrib/debian/copyright`.

laanwj commented:
@MarcoFalke Looks good to me - though I think replacing the contents of assets-attrribution with a link to the debian copyright file would be even better. `contrib/debian` is not the most straightforward place to look for people that are suddenly missing the file.

Commit:
[doc] Merge doc/assets-attribution.md into contrib/debian/copyright

Even though the SVG source files are never used in the QT client,
their authors are mentioned in contrib/debian/copyright due to
simplicity.

MarcoFalke commented:
@laanwj Done.
